content/guides/production-build.md

```diff
@@ -167,21 +167,21 @@ module.exports = function() {
             sourceMapFilename: '[name].map'
         },
         resolve: {
-            extensions: ['', '.ts', '.js', '.json'],
+            extensions: ['.ts', '.js', '.json'],
             modules: [path.join(__dirname, 'src'), 'node_modules']
 
         },
         module: {
-            loaders: [{
+            rules: [{
                 test: /\.ts$/,
-                loaders: [
+                use: [
                     'awesome-typescript-loader',
                     'angular2-template-loader'
                 ],
                 exclude: [/\.(spec|e2e)\.ts$/]
             }, {
                 test: /\.css$/,
-                loaders: ['to-string-loader', 'css-loader']
+                use: ['to-string-loader', 'css-loader']
             }, {
                 test: /\.(jpg|png|gif)$/,
                 loader: 'file-loader'
```

Comment:
This like could become `use` too (`use: 'file-loader'`). Feel free to add yourself to page authors at metadata too.